linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/css/dashboard.css

```diff
@@ -74,3 +74,53 @@ nav ul.navbar-nav {
 .dentry .dentry-part:hover {
   background-color: #222;
 }
+
+.router div {
+  overflow: scroll;
+}
+
+/*
+* Router summary
+*/
+
+.router-summary {
+  clear: both;
+}
+
+.router-summary div {
+  padding: 10px;
+}
+
+.router-header {
+  float: left;
+  text-align: left;
+}
+
+.router-data {
+  float: right;
+}
+
+.router-data span {
+  float: left;
+  margin: 0px 30px;
```

Comment:
Tiny tiny nit: `margin: 0 30px;` is more consistent with 0-values throughout the rest of this file.